code-reviews/example3/adventure_game.py

```diff
@@ -0,0 +1,79 @@
+# An interactive adventure game!
+import random
+
+def print_welcome():
+    print("Welcome to the adventure game! 🛡")
+    print("Maintain your health while finding food 🥕")
+    print("But watch out for the wild animals! 🦊")
+    print("")
+
+def print_menu():
+    print("1. Explore")
+    print("2. Check status")
+    print("3. Rest")
+    print("4. Quit game")
+
+def explore():
+    global health
+    global food
+    event = random.randint(1, 3)
+    if event == 1:
+        print("You found a berry bush! +5 food")
+        food += 5
+    elif event == 2:
+        print("You were attacked by a wild animal! -5 health")
+        health -= 5
+    elif event == 3:
+        print("You found a small creek! +10 health")
+        health += 10
+    else:
+        print("You tripped and fell into a hole! -10 health")
+        health -= 10
+    if health > 100:
+        health = 100
+    elif health < 0:
```

Comment:
shouldn't be an else instead of an elif?

Reply:
short form ?

Reply:
Not sure what you mean by short form, could you explain?